src/Components/Dashboards/FacultyDashboard/FacultyDashboard.js

```diff
@@ -39,7 +42,19 @@ function FacultyDashboard(props) {
   const [selectedEquipmentFilter, setSelectedEquipmentFilter] = useState(null);
 
   // State for selected reservation
+  const [reservations, setReservations] = useState([]);
   const [selectedReservation, setSelectedReservation] = useState(null);
+  const [selectedReservationDetails, setSelectedReservationDetails] = useState([]);
+
+  const [isMyReservation, setIsMyReservation] = useState(false);
```

Comment:
Does this belong with the rest of the selected reservation states?

Reply:
done.